javascript-injectors/hello-world/pages/index.html

```diff
@@ -14,31 +14,26 @@
         <link href="https://fonts.googleapis.com/css?family=Inter" rel="stylesheet">
         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
 
-        <link href="../../theme/playground-theme.css" rel="stylesheet">
-        <link href="./assets/styles/main.css" rel="stylesheet">
+        <link href="../../../bootstrap/theme/playground-theme.css" rel="stylesheet">
```

Comment:
This doesn't look right to me. If we are copying in the `boostrap` folder in the build process, why do we need a relative path?

Reply:
@vpetersson This is for dev. While developing locally we just reference the original source.

Reply:
ah right - hence the `sed`. It's fine for now but feels a bit hackish.